packages/xml-lsp-server/test/server.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from "chai";
+
+import { validateDocument, SYNTAX_ERROR_MSG } from "../src/serverHelper";
+
+import {
+  TextDocument,
+  Diagnostic,
+  DiagnosticSeverity,
+  Position
+} from "vscode-languageserver";
+
+describe("Test XML Language Server", () => {
+  it("Test parsing error validation in xml document", async () => {
+    const doc: TextDocument = createTextDocument("xml", ">");
+    const pos: Position = { line: 0, character: 0 };
+    const diagnostic: Diagnostic = {
```

Comment:
`diagnostic` variable is redundant as confusing as we also have `diagnostic**s**`

I'd recommend to in-line it in:
```typescript
    const expectedDiagnostics: Diagnostic[] = [diagnostic];
```